src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryLayout.java

```diff
@@ -493,6 +496,50 @@ default VarHandle varHandle(Class<?> carrier, PathElement... elements) {
                 Set.of(), elements);
     }
 
+    /**
+     * Creates a method handle which, given a memory segment, returns a {@link MemorySegment#asSlice(long,long)}
```

Comment:
```suggestion
     * Creates a method handle which, given a memory segment, returns a {@link MemorySegment#asSlice(long,long) slice}
```

Reply:
I tried this, but it doesn't seem to show up in the javadoc

Reply:
Ah, no wait it does. I was looking in the wrong place :)